source/Nevermore/Advanced/ReadTransaction.cs

```diff
@@ -90,15 +90,15 @@ public void Open()
 
             connection = connectionFactory(registry.ConnectionString);
             connection.OpenWithRetry();
-            
+
             TransactionTimer = new TimedSection(ms => configuration.TransactionLogger.Write(ms, name));
         }
 
-        public async Task OpenAsync()
+        public async Task OpenAsync(CancellationToken cancellationToken = default)
```

Comment:
This is an ABI break and could cause a crash if an old DLL indirectly gets handed a new version of Nevermore (e.g. multiple projects in the same solution reference different versions).
I don't expect we care about such things for Nevermore, just felt I should raise it just in in case.